packages/esm-appointments-app/src/config-schema.ts

```diff
@@ -47,6 +48,16 @@ export const configSchema = {
     _description: 'Display for the privilege to view and manage appointments',
```

Comment:
Name of the privilege to display the full view of the Appointments dashboard widget.